tools/firmware-utils/src/ptgen.c

```diff
@@ -132,20 +209,73 @@ static inline unsigned long round_to_kb(long sect) {
         return ((sect - 1) / kb_align + 1) * kb_align;
 }
 
+/* Compute a CRC for guid partition table */
+static inline unsigned long gpt_crc32(void *buf, unsigned long len)
+{
+	return cyg_crc32_accumulate(~0L, buf, len) ^ ~0L;
+}
+
+/* Parse a guid string to guid_t struct */
+static inline int guid_parse(char *buf, guid_t *guid)
```

Comment:
Not worth changing, but the compiler is generally pretty good at detecting what's worth making an inline.  Not sure this is necessary.